assets/src/pages/webrtcInternals/parsIncomingStats.tsx

```diff
@@ -0,0 +1,91 @@
+import moment from "moment";
+import { MAX_DATA_POINTS_ON_CHART } from "../room/consts";
+import { tail } from "ramda";
+
+export type Section = {
+  key: string;
+  descriptive: DescriptiveValue[];
+  charts: ChartEntry[];
+  subsections?: { [name: string]: Section };
+};
+
+type DescriptiveValue = {
+  name: string;
+  value: string;
+};
+
+type ChartEntry = {
+  chartTitle: string;
+  xs: number[];
+  ys: number[];
+};
+
+type ChannelInputValue = string | number | ChannelInput | null;
+
+type ChannelInput = {
+  [key: string]: ChannelInputValue;
+};
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const isChannelInput = (stats: any): stats is ChannelInput => {
+  return stats && typeof stats === "object" && Object.keys(stats).length > 0;
+};
```

Comment:
You could remove eslint comment if you change `any` to `unknown` and double negate return value. 

```suggestion
export const isChannelInput = (stats: unknown): stats is ChannelInput => {
  return !!(stats && typeof stats === "object" && Object.keys(stats).length > 0);
};
```